posthog/models/team/team.py

```diff
@@ -90,6 +93,21 @@ def get_team_from_token(self, token: Optional[str]) -> Optional["Team"]:
         except Team.DoesNotExist:
             return None
 
+    def get_team_from_token_or_cache(self, token: Optional[str]) -> Optional["Team"]:
```

Comment:
nit but possibly important: Can we name this `get_team_from_cache_or_token` as that is what it is actually doing.

Reply:
makes sense